content/en/docs/concepts/storage/volumes.md

```diff
@@ -1239,6 +1239,20 @@ Learn how to
 For more information on how to develop a CSI driver, refer to the [kubernetes-csi
 documentation](https://kubernetes-csi.github.io/docs/)
 
+#### Migrating to CSI drivers from in-tree plugins
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+The CSI Migration feature, when enabled, directs all operations against existing in-tree
```

Comment:
to be completely accurate, not *all* operations *yet* :) Just Attach/Detach/Mount/Unmount/Create/Delete.

No resize, block, inline, others?

Reply:
Agree. I think each of those will be under their respective feature state flags as they come in.